tests/kernel/copy_file_test.cpp

```diff
@@ -90,7 +90,7 @@ void punch_holes(int fd, bool hole_at_start = false, bool hole_at_end = false);
 /**
  * Convenience wrapper around ::mmap
  */
-std::pair<char const *, off_t> mmap(int fd);
+std::pair<unsigned char const *, off_t> mmap(int fd);
```

Comment:
Please document the interface. might std::byte also be a good type for the pair's first?